cmd/unifyLabels.go

```diff
@@ -0,0 +1,209 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/google/go-github/v39/github"
+	"github.com/spf13/cobra"
+)
+
+func getLabelModifier(fix bool, repo string, githubClient *github.Client) LablesModifier {
```

Comment:
I got thinking, regarding testing. Could it be useful, to encapsulate the githubClient into a classed owned by us, such that you can mock it for tests.